src/main/kotlin/no/nav/klage/oppgave/repositories/KafkaVedtakEventRepository.kt

```diff
@@ -0,0 +1,11 @@
+package no.nav.klage.oppgave.repositories
+
+import no.nav.klage.oppgave.domain.klage.KafkaVedtakEvent
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
+import java.util.*
+
+interface KafkaVedtakEventRepository : JpaRepository<KafkaVedtakEvent, UUID> {
+    @Query("SELECT k FROM KafkaVedtakEvent k WHERE k.status <> 'SENDT'")
```

Comment:
Denne ville jeg prøvd å skrive som en metode som Spring Data kan generere opp for deg, det bør være mulig

Reply:
Kan også testes gjennom "repositoryTest" som vi har et par fra før.